src/services/info/location-info.ts

```diff
@@ -0,0 +1,109 @@
+import { urlStorage } from '@/services/state-persistor/storage/url-storage'
+import { transform } from 'ol/proj'
+import { Coordinate } from 'ol/coordinate'
+import { Projection } from 'ol/proj'
+
+export const INFO_PROJECTIONS = {
+  'EPSG:2169': 'Luref',
+  'EPSG:4326': 'Lon/Lat WGS84',
+  'EPSG:4326:DMS': 'Lon/Lat WGS84 DMS',
+  'EPSG:4326:DMm': 'Lon/Lat WGS84 DM',
+  'EPSG:3263*': 'WGS84 UTM',
+}
+
+const HEMISPHERES = ['EW', 'NS']
+
+export async function queryInfo(location: Coordinate, fromCrs: Projection) {
+  return {
+    shortUrl: await create_short_url(location),
+    clickCoordinateLuref: transform(location, fromCrs, 'EPSG:2169'),
+    formatted_coordinates: Object.fromEntries(
+      Object.entries(INFO_PROJECTIONS).map(([crs, label]) => [
+        label,
+        formatCoords(location, fromCrs, crs),
+      ])
+    ),
+  }
+}
+
+export function formatCoords(
```

Comment:
We have a helper (or utility) to format data to string, this should be move in the helper.

Reply:
done